common/app/model/facia/PressedCollection.scala

```diff
@@ -30,6 +30,8 @@ case class PressedCollection(
     config: CollectionConfig,
     hasMore: Boolean,
     targetedTerritory: Option[TargetedTerritory],
+    liteCuratedLength: Option[Int],
+    liteBackfillLength: Option[Int],
```

Comment:
Just a thought: What if we had `visible` here instead of `liteCuratedLength` and `liteBackfillLength`? This way we could calculate the lengths when we need them, e.g. in `FaciaController` `renderShowMore` method and avoid the double pattern matching we're currently doing.

Reply:
We considered this option when writing it initially, but I think @jamesgorrie had reasons for preferring to store the two numbers separately. Do you remember your reasoning on this point James?